standards-catalogue/adr/0001-remove-tech-docs-gem-dependency.md

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
This directory should be removed for the time being. When we need to start writing decisions specifically for the standards catalogue we can create a new one?